news/4.34/jdt.html

```diff
@@ -134,6 +134,22 @@ <h2>Java Views and Dialogs</h2>
 			<img src="images/move-filtersbutton-to-top-other-way.png" alt="old menu">
 		  </p>
       </td>
+	  <tr id="experimental-preference">
+	  	  <td class="title"><strong>Experimental!</strong> Preference added to enable update on runtime behavior</a></td>
```

Comment:
The title should clearly state that this is about rescaling / multi-monitor HiDPI support. It should also state that this is about Windows. I propose to put the "experimental" in parenthesis at the end and not start the title with it, as it might unnecessary prevent people from reading the entry and trying it out.